test/unit/specs/components/ActivityCollector/attachClickActivityCollector.spec.js

```diff
@@ -10,5 +10,45 @@ OF ANY KIND, either express or implied. See the License for the specific languag
 governing permissions and limitations under the License.
 */
 
-// eslint-disable-next-line no-unused-vars
 import attachClickActivityCollector from "../../../../../src/components/ActivityCollector/attachClickActivityCollector";
+
+describe("ActivityCollector::attachClickActivityCollector", () => {
+  const cfg = {
+    clickCollectionEnabled: true
+  };
+  const mockEventManager = {
+    createEvent: () => {
+      return {
+        isEmpty: () => true
+      };
+    }
+  };
+  const mockLifeCycle = {
+    onClick: () => {
+      return Promise.resolve();
+    }
+  };
+  let clickHandler;
+  beforeEach(() => {
+    // eslint-disable-next-line no-unused-vars
+    spyOn(document, "addEventListener").and.callFake((name, handler, type) => {
+      clickHandler = handler;
+    });
+  });
+
+  fit("Attaches click handler if clickCollectionEnabled is set to true", () => {
+    attachClickActivityCollector(cfg, mockEventManager, mockLifeCycle);
+    expect(document.addEventListener).toHaveBeenCalled();
+  });
+  fit("Does not attach click handler if clickCollectionEnabled is set to false", () => {
+    cfg.clickCollectionEnabled = false;
```

Comment:
I messaged Martin about this already.  You need to add the cfg setup to the beforeEach so that the run-order of the tests doesn't matter.